Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -765,14 +763,9 @@ private void ToggleFlowLauncher()
 
         public void Save()
         {
-            if (!_saved)
-            {
-                _historyItemsStorage.Save();
-                _userSelectedRecordStorage.Save();
-                _topMostRecordStorage.Save();
-
-                _saved = true;
-            }
+            _historyItemsStorage.Save();
+            _userSelectedRecordStorage.Save();
+            _topMostRecordStorage.Save();
```

Comment:
I remove this because it will prevent flow to continue save the records once user hit "Save Settings"